astro/src/content/docs/apis/_api-key-cross-tenant-note.mdx

```diff
@@ -1 +1 @@
-Tenant scoped keys can retrieve configuration for FusionAuth entities such as identity providers and lambdas that may be shared between tenants. Limit the API key by specifying required endpoints and permissions as well as the tenant.
+Tenant scoped keys can retrieve configuration for FusionAuth entities such as lambdas that may be shared between tenants. Limit the API key by specifying required endpoints and permissions as well as the tenant.
```

Comment:
```suggestion
Tenant scoped keys can retrieve configuration for FusionAuth entities such as global identity providers and lambdas that may be shared between tenants. Limit the API key by specifying required endpoints and permissions as well as the tenant.
```
[optional]: probably OK to remove identity provider mentions entirely, but I think it would be accurate to say 'global identity providers' as well. just wanted to provide another option.